AdornUI-Stylesheets/alert.css

```diff
@@ -0,0 +1,32 @@
+.alert-box {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  width: auto;
+  margin: 1rem;
+  padding: 1rem;
+  font-size: 2rem;
+  border-radius: 5px;
+  font-family: "EB Garamond", serif;
+}
+.close-alert {
+  float: right;
```

Comment:
you can use flex properties for placing your items in container or margin like this
```css
margin-left:auto;
```
instead of using float